src/codec_dav1d.c

```diff
@@ -57,8 +57,8 @@ static avifBool dav1dCodecGetNextImage(struct avifCodec * codec,
 {
     if (codec->internal->dav1dContext == NULL) {
         // Give all available threads to decode a single frame as fast as possible
-        codec->internal->dav1dSettings.n_frame_threads = 1;
-        codec->internal->dav1dSettings.n_tile_threads = AVIF_CLAMP(decoder->maxThreads, 1, DAV1D_MAX_TILE_THREADS);
+        codec->internal->dav1dSettings.n_threads = 1;
+        codec->internal->dav1dSettings.max_frame_delay = AVIF_CLAMP(decoder->maxThreads, 1, 256);
```

Comment:
You might want to use
```c
#if defined(DAV1D_API_VERSION_MAJOR) && DAV1D_API_VERSION_MAJOR >= 6
// new fields and code here
#else
// old code here
#endif
```
in order to keep compile compatibility with older dav1d versions and installs

Reply:
OK. I added DAV1D_API_VERSION_MAJOR

Reply:
This can be marked as resolved